evalml/tests/automl_tests/test_automl.py

```diff
@@ -83,7 +83,7 @@ def test_search_results(X_y_regression, X_y_binary, X_y_multi, automl_type, obje
         expected_pipeline_class = MulticlassClassificationPipeline
         X, y = X_y_multi
 
-    automl = AutoMLSearch(X_train=X, y_train=y, problem_type=automl_type, objective=objective, max_iterations=2, n_jobs=1)
+    automl = AutoMLSearch(X_train=X, y_train=y, problem_type=automl_type, objective=objective, optimize_thresholds=False, max_iterations=2, n_jobs=1)
```

Comment:
needed to set `optimize_thresholds` as False in order to avoid issues during training `best_pipeline` with `predict_proba`. Otherwise, I could have mocked `predict_proba` for all the tests but this was a faster approach with similar results.